packages/@aws-cdk/aws-cloudfront/lib/experimental/edge-function.ts

```diff
@@ -1,12 +1,13 @@
-import * as crypto from 'crypto';
 import * as path from 'path';
 import * as cloudwatch from '@aws-cdk/aws-cloudwatch';
 import * as ec2 from '@aws-cdk/aws-ec2';
 import * as iam from '@aws-cdk/aws-iam';
 import * as lambda from '@aws-cdk/aws-lambda';
+// hack, as this is not exported by the Lambda module
+import { calculateFunctionHash } from '@aws-cdk/aws-lambda/lib/function-hash';
```

Comment:
Why not export then instead of reaching internals?

Reply:
Less surface area for the module?